user-storage-simple/src/main/java/org/keycloak/quickstart/writeable/PropertyFileUserStorageProvider.java

```diff
@@ -273,7 +252,8 @@ public boolean updateCredential(RealmModel realm, UserModel user, CredentialInpu
 
     @Override
     public void disableCredentialType(RealmModel realm, UserModel user, String credentialType) {
-        if (!credentialType.equals(CredentialModel.PASSWORD)) return;
+        if (!credentialType.equals(PasswordCredentialModel.TYPE))
+            return;
```

Comment:
please revert formatting changes (and keep `CredentialModel.PASSWORD` -> `PasswordCredentialModel.TYPE`)